openapi.yaml

```diff
@@ -60,6 +60,45 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
 
+  /validate:
+    post:
+      summary: Validate the given AsyncAPI document.
+      operationId: validate
+      tags:
+        - validate
+        - template
+      requestBody:
+        description: Validate the given AsyncAPI document with the AsyncAPI parser.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Template'
```

Comment:
I guess we still need to create the proper schema for the payload, right?

Reply:
yes, I named it `ValidateDocument`, let me know if it's okay 👍